### PR TITLE
Pytorchmodel

### PR DESCRIPTION
**Pull request by G4V**

### CODE REVIEW

File: package/model-pytorch-rnnt/.cm/meta.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "model_file": "rnnt.pt",
+    "model_id": "rnnt",
+    "no_os_in_suggested_path": "yes",
+    "no_ver_in_suggested_path": "yes",
+    "version": "20200211"
+  },
+  "end_full_path": {
+    "linux": "rnnt.pt"
+  },
+  "package_extra_name": " (RNN-T)",
```
Comment (psyhtest) [inline package/model-pytorch-rnnt/.cm/meta.json]: (MLPerf RNN-T)?

File: program/mlperf-inference-v0.7-rnnt/.cm/meta.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  "run_deps": {
+    "dataset": {
+      "local": "yes",
+      "name": "Librispeech Dev Clean dataset",
+      "sort": 20,
+      "tags": "dataset,librispeech,preprocessed,wav,dev-clean"
+    },
+    "lib-python-loadgen": {
+      "local": "yes",
+      "name": "Python loadgen library",
```
Comment (psyhtest) [inline program/mlperf-inference-v0.7-rnnt/.cm/meta.json]: LoadGen

File: program/mlperf-inference-v0.7-rnnt/.cm/meta.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      "sort": 60,
+      "tags": "lib,python-package,loadgen"
+    },
+    "mlperf-inference-dividiti-rnnt": {
+      "local": "yes",
+      "name": "Python opencv library",
```
Comment (psyhtest) [inline program/mlperf-inference-v0.7-rnnt/.cm/meta.json]: MLPerf Inference source

File: program/mlperf-inference-v0.7-rnnt/.cm/meta.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    "CK_RNNT_ACCURACY": ""
+  },
+  "tags": [
+    "speech-recognition",
+    "pytorch",
+    "rnnt",
+    "rnn-t",
+    "mlperf"
+  ],
+  "template": "yes",
```
Comment (psyhtest) [inline program/mlperf-inference-v0.7-rnnt/.cm/meta.json]: Please remove the `template` and `template_type` keys.

File: package/model-pytorch-rnnt/README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# RNN-T package
+
+```
+$ ck clean env --tags=model,pytorch,rnnt -f
+$ ck install package:model-pytorch-rnnt
+
```
Comment (psyhtest) [inline package/model-pytorch-rnnt/README.md]: Remove blank line.

File: package/model-pytorch-rnnt/README.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# RNN-T package
+
+```
+$ ck clean env --tags=model,pytorch,rnnt -f
+$ ck install package:model-pytorch-rnnt
```
Comment (psyhtest) [inline package/model-pytorch-rnnt/README.md]: Why not use `--tags` as above.

File: program/mlperf-inference-v0.7-rnnt/.cm/meta.json (hunk 4 above)
Comment (G4V) [reply program/mlperf-inference-v0.7-rnnt/.cm/meta.json]: Done

File: package/model-pytorch-rnnt/README.md (hunk 5 above)
Comment (G4V) [reply package/model-pytorch-rnnt/README.md]: Done

File: package/model-pytorch-rnnt/README.md (hunk 6 above)
Comment (G4V) [reply package/model-pytorch-rnnt/README.md]: Done - a cut and paste from the previous readme, if I'm honest.

**Pull request merged**